medicines/doc-index-updater/src/document_manager/mod.rs

```diff
@@ -67,12 +72,16 @@ async fn delete_document_handler(
 ) -> Result<JobStatusResponse, Rejection> {
     if let Ok(mut queue) = delete_factory().await {
         let id = accept_job(&state_manager).await?.id;
+        let correlation_id = id.to_string();
+        let correlation_id = correlation_id.as_str();
 
         let message = DeleteMessage {
             job_id: id,
             document_content_id,
         };
-        queue_job(&mut queue, &state_manager, message).await
+        queue_job(&mut queue, &state_manager, message)
+            .instrument(tracing::info_span!("document_manager", correlation_id))
```

Comment:
:+1:

Would be nice to do the same in create document handler.

Reply:
true, this is like never ending story :)

Reply:
Done